dashboard/db/migrate/20150128022616_create_ops_tables.rb

```diff
@@ -9,7 +9,7 @@ def change
       t.timestamps
     end
 
-    create_table :cohorts do |t|
+    create_table :cohorts do |t|  # rubocop:disable Style/SymbolProc
```

Comment:
This [disables](http://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code) the cop only for the line it is on (as it is an end-of-line comment).